### ISSUE TITLE
Missing stopforumspam parameters?

### ISSUE THREAD

**Issue by boomzillawtf** [akhoury/nodebb-plugin-spam-be-gone#70]
Sometimes when I try to report a spammer to stopforumspam using the little flag button, instead of reporting, I get this:
![image](https://user-images.githubusercontent.com/13367628/70532149-72fa4b80-1b24-11ea-97ea-ada9c3919ce9.png)

And in my log:
```
error: [plugins/spam-be-gone][report-error] You must have all search parameters for StopForumSpam.com to accept your submission. {"email":"","username":"[[global:guest]]"}

```

What parameters does it think it missing? The email and IP and username are all right there in the queue.

**Comment by nefarius** [comment]
Same, I can not report at all, same message every time:

![image](https://user-images.githubusercontent.com/286631/99067629-545dc980-25ab-11eb-855e-d74d387d8dbe.png)

**Comment by barisusakli** [comment]
Yeah this looks bugged, the report user function only works for actual user accounts. The entries in the registration queue dont have accounts yet so it doesnt work there. I will work on a fix.

**Comment by barisusakli** [comment]
published 0.7.4, let me know if resolved now
